### ISSUE TITLE
Camera initialization error

### ISSUE THREAD

**Issue by tkaufthe** [nzapponi/homebridge-simplisafe3#106]
Long time reader, first time writer. 

After the latest update where the camera's are handled differently, I am no longer able to see the camera. the error I see in my log is as follows:

5/12/2020, 5:29:19 PM [Home Alarm] An error occurred while refreshing accessories
5/12/2020, 5:29:19 PM [Home Alarm] TypeError: this.accessory.configureController is not a function
    at SS3SimpliCam.setAccessory (/home/hoobs/.hoobs/node_modules/homebridge-simplisafe3/accessories/simplicam.js:97:20)
    at SS3Platform.refreshAccessories (/home/hoobs/.hoobs/node_modules/homebridge-simplisafe3/index.js:357:31)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)

Here is also the config i have for my simplisafe (private info redacted)

{
            "platform": "homebridge-simplisafe3.SimpliSafe 3",
            "plugin_map": {
                "plugin_name": "homebridge-simplisafe3"
            },
            "name": "Home Alarm",
            "cameras": true,
            "sensorRefresh": 15,
            "persistAccessories": false,
            "resetSimpliSafeId": false,
            "auth": {
                "username": "xxxxxxxx@xxx.com",
                "password": "XXXXXX"
            },
            "cameraOptions": {
                "ffmpegPath": "/usr/local/bin/ffmpeg",
                "videoOptions": "-vcodec h264_omx -tune false -preset false",
                "sourceOptions": "-vcodec h264_mmal"
            },
            "debug": false
        }

the camera options i am using were put in there as a test. I get the same results with or without them. Additionally, the camera appeared and was working prior to the 1.5.0 update and I have 3 other cameras (wyze) that work using the ffmpeg codec.

Please note, i am fairly new at this, so as i read other posts where an entire log is attached, I am not sure how to display mine like that other than what is in the hoobs window.

My hardware is a pi 4 running hoobs 3.something (whatever the latest version is) and the only accessories i connect to homebridge is the simplisafe and the wyze.

All my other accessories for simplisafe are fine like door sensors, smoke detectors and even setting the alarm, it just the doorbell cam.

Any thought would be greatly appreciated.

**Comment by nikonratm** [comment]
HI @tkaufthe , my guess is you are not running the correct version of Homebridge. As noted in the release notes, this version of the plugin requires at least version 1.0.0 of Homebridge which is a major update. I am not sure if HOOBS has been updated to this, that would be a question for HOOBS. If not, you can down-grade your plugin. Again, see [the README](https://github.com/nzapponi/homebridge-simplisafe3/blob/master/README.md)

**Comment by tkaufthe** [comment]
Thank you very much for the quick reply. I must have read that part of the readme too quickly. I was able to roll back to a previous version of the plugin and it worked. 

I then built a new image just based on the Homebridge build, and installed the plugins. That also worked very well. So good in fact, I'll be staying on Homebridge rather than hoobs.. Its just lighter and the mobile gui is much easier to maintain.

Thanks for all the hard work on this plugin.
